interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -161,7 +161,7 @@ def test_setCompositionDict(self):
         self.assertNear(Y[3], 0.75)
 
     def test_getCompositionDict(self):
-        self.phase.X = 'oh:1e-9, O2:0.4, AR:0.6'
+        self.phase.X = 'OH:1e-9, O2:0.4, AR:0.6'
```

Comment:
I would prefer to leave these tests as-is -- they are written using different cases to confirm that case insensitive matching is working.